src/components/LoginRegisterIntroduction/LoginRegisterPopUp/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,29 @@
+
+import { useState } from "react";
+import CredentialsFieldsComponent from "./BaseFields/CredentialsFieldsComponent";
+import PersonalFieldsComponent from "./BaseFields/PersonalFieldsComponent";
+
+const RegisterForm = () => {
+  const [numberPart, setNumberPart] = useState(1)
```

Comment:
`numberPart` name does not really tell us what is it or what setNumberPart does. Maybe you whould think of a clearer name, for example `registrationStep` ? You can try asking Chat GPT for name suggestions whenever you find yourself struggling to come up with one.